packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4199,9 +4200,12 @@ class IndexedStack extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final List<Widget> wrappedChildren = List<Widget>.generate(children.length, (int i) {
-      return Visibility.maintain(
-        visible: i == index,
-        child: children[i],
+      return ExcludeFocus(
```

Comment:
This feels something should be added to Visibility widget.

Reply:
Also it seems it should not use Visibility.maintain since the maintain by default allows interactivity and semantics and animation unecessarily it should use
```dart
Visibility(
  visible: i == index,
  maintainSize: true,
  maintainState: true,
  child: children[i]
)
```

Reply:
Thanks for the feedback! Fixed the issue in Visibility widget instead. The new solution makes Visibility ignore focus when child is hidden for all combinations of maintain-parameters. I'm a bit unsure if it should be ignored when `maintainInteractivity` is true, what do you think?

Removed maintain interactivity and semantics from indexed stack and added a test. Could not remove `maintainAnimation` since that is needed if `maintainSize` is set.

Reply:
It should ignored when maintainInteractivity is true since focus is part of interaction